### DISCUSSION TITLE
Terms of Use on jw.org

### DISCUSSION THREAD

**Discussion by FussuChalice** [sircharlo/meeting-media-manager#2181] [Q&A] [answered]
Hello! On the official website of Jehovah's Witnesses, the rules of use say that you cannot: Create for distribution purposes, any software applications, tools, or techniques that are specifically made to collect, copy, download, extract, harvest, or scrape data, HTML, images, or  text from this site.  (This does not prohibit the distribution of free, non-commercial applications designed to download electronic files such as EPUB, PDF, MP3, and MP4 files from public areas of this site.)
 https://www.jw.org/en/terms-of-use/#link2

 Maybe I misunderstood you, but you created a script for downloading media files from this site.

**Comment by mtdvlpr** [comment, marked as answer]
Hi @FussuChalice, note the `This does not prohibit the distribution of free, non-commercial applications designed to download electronic files such as EPUB, PDF, MP3, and MP4 files from public areas of this site.`

That's what we do. We made a free, non-commercial application that downloads the meeting files from public areas of the site.

**Reply by FussuChalice** [reply]
Then I apologize. In that case, can I join your project? If yes, then give me the task.

**Reply by mtdvlpr** [reply]
Of course, all help is welcome! Please see #722 for information on how to contribute. We're currently busy with a major refactor, so no new features for now.

**Reply by FussuChalice** [reply]
Thanks!
